x-pack/plugins/uptime/public/components/monitor/synthetics/step_detail/waterfall/data_formatting.test.ts

```diff
@@ -29,463 +183,327 @@ describe('Palettes', () => {
-      status: 200,
-      mimeType: 'application/javascript',
-      requestSentTime: 18098833.537,
-      requestStartTime: 18098837.233999997,
-      loadEndTime: 18098977.648000002,
-      timings: {
-        blocked: 84.54599999822676,
-        receive: 3.068000001803739,
-        queueing: 3.69700000010198,
-        proxy: -1,
-        total: 144.1110000014305,
-        wait: 52.56100000042352,
-        connect: -1,
-        send: 0.2390000008745119,
-        ssl: -1,
-        dns: -1,
-      },
-    },
-  ];
-
-  const networkItemsWithoutFullTimings: NetworkItems = [
-    networkItems[0],
-    {
-      timestamp: '2021-01-05T19:22:28.928Z',
-      method: 'GET',
-      url: 'file:///Users/dominiqueclarke/dev/synthetics/examples/todos/app/app.js',
-      status: 0,
-      mimeType: 'text/javascript',
-      requestSentTime: 18098834.097,
-      loadEndTime: 18098836.889999997,
-      timings: {
-        total: 2.7929999996558763,
-        blocked: -1,
-        ssl: -1,
-        wait: -1,
-        connect: -1,
-        dns: -1,
-        queueing: -1,
-        send: -1,
-        proxy: -1,
-        receive: -1,
-      },
-    },
-  ];
-
-  const networkItemsWithoutAnyTimings: NetworkItems = [
-    {
-      timestamp: '2021-01-05T19:22:28.928Z',
-      method: 'GET',
-      url: 'file:///Users/dominiqueclarke/dev/synthetics/examples/todos/app/app.js',
-      status: 0,
-      mimeType: 'text/javascript',
-      requestSentTime: 18098834.097,
-      loadEndTime: 18098836.889999997,
-      timings: {
-        total: -1,
-        blocked: -1,
-        ssl: -1,
-        wait: -1,
-        connect: -1,
-        dns: -1,
-        queueing: -1,
-        send: -1,
-        proxy: -1,
-        receive: -1,
-      },
-    },
-  ];
-
-  const networkItemsWithoutTimingsObject: NetworkItems = [
-    {
-      timestamp: '2021-01-05T19:22:28.928Z',
-      method: 'GET',
-      url: 'file:///Users/dominiqueclarke/dev/synthetics/examples/todos/app/app.js',
-      status: 0,
-      mimeType: 'text/javascript',
-      requestSentTime: 18098834.097,
-      loadEndTime: 18098836.889999997,
-    },
-  ];
-
-  const networkItemsWithUncommonMimeType: NetworkItems = [
-    {
-      timestamp: '2021-01-05T19:22:28.928Z',
-      method: 'GET',
-      url: 'https://unpkg.com/director@1.2.8/build/director.js',
-      status: 200,
-      mimeType: 'application/x-javascript',
-      requestSentTime: 18098833.537,
-      requestStartTime: 18098837.233999997,
-      loadEndTime: 18098977.648000002,
-      timings: {
-        blocked: 84.54599999822676,
-        receive: 3.068000001803739,
-        queueing: 3.69700000010198,
-        proxy: -1,
-        total: 144.1110000014305,
-        wait: 52.56100000042352,
-        connect: -1,
-        send: 0.2390000008745119,
-        ssl: -1,
-        dns: -1,
-      },
-    },
-  ];
+  beforeEach(() => {
+    mockMoment();
+  });
 
-  it('formats timings', () => {
+  it('formats series timings', () => {
```

Comment:
We'll want to think about a way to make these tests friendlier in the future, but since they already exist I'm ok with not updating them as part of this PR as it already has a really large scope. Follow-up issue: https://github.com/elastic/uptime/issues/297